mail_activity_reminder/tests/test_mail_activity_reminder.py

```diff
@@ -0,0 +1,197 @@
+# Copyright 2020 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from dateutil.relativedelta import relativedelta
+from pytz import timezone, UTC
+
+from odoo import fields
+from odoo.tests import common
+
+
+class TestMailActivityReminder(common.TransactionCase):
```

Comment:
pls use a SavepointCase and move all global setup to setUpClass instead of setUp.
Also:
* disable tracking globally + reset pwd as you create users, w/ `cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True, no_reset_password=True))`
* call `self.MailActivityType.create` on setUpClass and in tests change only the field you need
* for testing dates, instead of using `mail_activity_reminder_now` freeze the time (unless you think `mail_activity_reminder_now` is something you plan to use functionally and not only for testing)